src/storage/storage_block_test.go

```diff
@@ -5,6 +5,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/0xb10c/bademeister-go/src/test"
+
```

Comment:
did go-imports add this blank line? seem weird to me..

Reply:
no, it probably was intellij

I'll add a commit to have pep8-style import order (stdlib/thirdparty/local)